bcipy/signal/model/keyinput.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+
+def compute_keyinput_probs(
+    inquiry: List[str],
+    symbol_set: List[str],
+    user_error_prob: float,
+    proceed: bool,
+):
+    """
+    When the user responds to an inquiry preview, we compute a likelihood term for each symbol
+    based on their response. These likelihood terms will be multiplied with the prior probabilities of
+    each symbol and then normalized, in order to update the symbol probabilities.
+
+    Args:
+        inquiry - the symbols presented in the inquiry preview
+        symbol_set - the full alphabet of symbols
+        proceed - whether or not the user wants to proceed with this inquiry
+        user_error_prob - the probability that the user selects incorrectly during preview
+    """
+    K = len(inquiry)
```

Comment:
no single letter varibles. This should fail linting.

Reply:
At some point I'll be sure to setup up autoformatting and linting to match project standards - not sure if there's a config file already available that I didn't notice